cmd/storagenode-updater/main.go

```diff
@@ -136,62 +149,64 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
-func update(ctx context.Context, nodeID storj.NodeID) (err error) {
-	client := checker.New(runCfg.ClientConfig)
+func update(ctx context.Context, binPath, serviceName string, renameBinary renameFunc) (err error) {
+	if nodeID.IsZero() {
+		log.Fatal("empty node ID")
+	}
 
-	currentVersion, err := binaryVersion(runCfg.BinaryLocation)
-	if err != nil {
-		return errs.Wrap(err)
+	var currentVersion version.SemVer
+	if serviceName == updaterServiceName {
+		// TODO: find better way to check this binary version
+		currentVersion = version.Build.Version
+	} else {
+		currentVersion, err = binaryVersion(binPath)
+		if err != nil {
+			return errs.Wrap(err)
+		}
 	}
 
+	client := checker.New(runCfg.ClientConfig)
 	log.Println("downloading versions from", runCfg.ServerAddress)
-	shouldUpdate, newVersion, err := client.ShouldUpdate(ctx, runCfg.ServiceName, nodeID)
+	shouldRollout, newVersion, err := client.ShouldUpdate(ctx, serviceName, nodeID)
```

Comment:
`shouldRollout` should be renamed back too.

Reply:
done